PowerEditor/installer/APIs/autoit.xml

```diff
@@ -0,0 +1,16099 @@
+<?xml version="1.0" encoding="Windows-1252" ?>
+<NotepadPlus>
+	<AutoComplete>
```

Comment:
Should language be added?

Reply:
which language?

Reply:
Other files have e.g.:
`
	<AutoComplete language="C">

`

Is there a specific reason why enoding is windows-1252 and not utf-8?